online-shop/src/app/service/LoginService.ts

```diff
@@ -0,0 +1,37 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from "@angular/core";
+import { Route } from '@angular/router';
+import { Observable,tap } from 'rxjs';
+import { User } from 'src/user';
+
+@Injectable({
+    providedIn: 'root'
+})
+
+export class LoginService {
+
+    ROOT_URL = "http://localhost:3000/";
```

Comment:
**suggestion:** you can keep the url inside the `environment.ts` file or you could use a [proxy](https://angular.io/guide/build#proxying-to-a-backend-server)